src/move.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
`move` and `resize` do not need to be objects. They should be functions on the selection object that update the state there.